testbed/tests/widgets/test_button.py

```diff
@@ -95,6 +95,10 @@ async def test_press(widget, probe):
 
 async def test_background_color_transparent(widget, probe):
     "Buttons treat background transparency as a color reset."
+    del widget.style.background_color
+    resetted_background_color = probe.background_color
```

Comment:
"resetted" isn't a word; suggest:
```suggestion
    original_background_color = probe.background_color
```